client/a8c-for-agencies/sections/referrals/referral-details/components/assigned-to.tsx

```diff
@@ -78,15 +88,103 @@ const AssignedTo = ( { purchase, handleAssignToSite, data, isFetching }: Props )
 		);
 	}
 
+	let tooltip = '';
+
+	let cancellationInfo = null;
+
+	// If the purchase has a subscription we can get the information from the object.
+	if ( purchase.subscription && ! purchase.subscription.is_refundable ) {
```

Comment:
I think we need to only show this message when a subscription is still active AND autorenew is off, rather than checking `is_refundable`. I think Jeff's mockup was a little off in that this wouldn't show next to subscriptions with the `Canceled` label since they're very much still active – the only difference is that they won't renew when that time comes.

Reply:
Here's an example of a subscription that's had autorenew disabled (still active) and where we'd want to add a tooltip.
<img width="954" height="134" alt="Screenshot 2025-09-04 at 4 43 20 PM" src="https://github.com/user-attachments/assets/43ec59c2-e7cf-4552-837c-e29c238911b3" />
<img width="1507" height="295" alt="Screenshot 2025-09-04 at 4 43 05 PM" src="https://github.com/user-attachments/assets/ff3b6a11-f883-4552-aaf3-b1588cab1b0f" />